packages/devtools_app/lib/src/screens/network/network_service.dart

```diff
@@ -82,9 +81,9 @@ class NetworkService {
   Future<void> _clearHttpProfile() async {
     assert(serviceManager.service != null);
     if (serviceManager.service == null) return;
-
-    await serviceManager.service.forEachIsolate((isolate) async {
-      final future = serviceManager.service.clearHttpProfile(isolate.id);
+    final service = serviceManager.service!;
```

Comment:
move this to the top of the method, remove the assert, and then check if the local var is null in the if statement